src/transform/type.ts

```diff
@@ -0,0 +1,45 @@
+import { ItemsObject, ParameterObject } from "types";
+import { tsUnionOf } from "../utils";
+
+function transformEnum(enumArray: string[]): string {
+  return tsUnionOf(enumArray.map((e) => (typeof e === "string" ? `'${e.replace(/'/g, "\\'")}'` : e)));
+}
+
+function transformItemsObj(obj: ItemsObject): string {
+  switch (obj.type) {
```

Comment:
This code actually already exists in [utils.ts#L29](https://github.com/drwpow/openapi-typescript/blob/main/src/utils.ts#L29). So by re-adding different logic, we could introduce bugs.